tools/tpm2_createek.c

```diff
@@ -136,14 +136,14 @@ struct alg_map {
 
 static const alg_map alg_maps[] = {
     { "rsa",           "rsa2048:aes128cfb", "sha256", &policy_a_sha256, ATTRS_A },
-    { "rsa2048",       "rsa2048:aes128cfb", "sha256", &policy_a_sha256, ATTRS_A },
+    { "rsa2048",       "rsa2048:aes128cfb", "sha256", &policy_b_sha256, ATTRS_B },
```

Comment:
Perhaps instead we want to introduce a `rsa2048_high` and `etc_nist_p256_high` alias?